devscripts/wine-py2exe.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -d wine-py2exe ]; then
 
     mkdir wine-py2exe
     cd wine-py2exe
-    export WINEPREFIX=`pwd`
+    WINEPREFIX=$(pwd); export WINEPREFIX
```

Comment:
Might have as well made it `"$(pwd)"`. If not just `"${PWD}"`.

Reply:
"" aren't needed for a '${}' or '$()' expansion that is the RHS of an assignment, according to POSIX, which is why Shellcheck doesn't suggest putting them in (even if the expansion happens to contain white space). Likely, `$PWD` would be equivalent, but that's not what was originally written.

Reply:
The `pwd` command uses the `${PWD}` by default, otherwise it asks the system.

^ Direct copypaste for `Bash 5.0.17(1)-release`:
```
$ help pwd
pwd: pwd [-LP]
    Print the name of the current working directory.

    Options:
      -L        print the value of $PWD if it names the current working
                directory
      -P        print the physical directory, without any symbolic links

    By default, `pwd' behaves as if `-L' were specified.

    Exit Status:
    Returns 0 unless an invalid option is given or the current directory
    cannot be read.
```
^ Here's the system-provided one's manpage (`man pwd`):
```
NAME
       pwd - print name of current/working directory

SYNOPSIS
       pwd [OPTION]...

DESCRIPTION
       Print the full filename of the current working directory.

       -L, --logical
              use PWD from environment, even if it contains symlinks

       -P, --physical
              avoid all symlinks

       --help display this help and exit

       --version
              output version information and exit

       If no option is specified, -P is assumed.

       NOTE: your shell may have its own version of pwd, which usually supersedes the version described here.  Please refer to your shell's documentation for details about the options it supports.

AUTHOR
       Written by Jim Meyering.

REPORTING BUGS
       GNU coreutils online help: <https://www.gnu.org/software/coreutils/>
       Report pwd translation bugs to <https://translationproject.org/team/>

COPYRIGHT
       Copyright © 2018 Free Software Foundation, Inc.  License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
       This is free software: you are free to change and redistribute it.  There is NO WARRANTY, to the extent permitted by law.

SEE ALSO
       getcwd(3)

       Full documentation at: <https://www.gnu.org/software/coreutils/pwd>
       or available locally via: info '(coreutils) pwd invocation'
```

So yeah, the `pwd` command can fail to find the path if your read+execute access to the folder was revoked just prior to the command or made unavailable by connection loss like in the case of NFS.

Which means it's just better to use the `pwd` command (or, if you wanna force the use of the system-provided one: `\pwd` or `command pwd`), and better yet, save the response like you're already doing, but also add a check in case it fails (I'm using `_current_folder` as an example):
```bash
# shellcheck disable=SC2155  ##We already check the emptiness.
[ ! "${_current_folder}" ] && {
    echo 'Failed to find the current working folder. Quitting.'

    exit 1
}
```